feature-libs/cart/base/root/events/cart.events.ts

```diff
@@ -72,6 +72,7 @@ export class CartAddEntrySuccessEvent extends CartEvent {
   entry?: OrderEntry;
   quantityAdded?: number;
   deliveryModeChanged?: boolean;
+  pickupStore?: string;
```

Comment:
Is this change _really_ related to this PR?

Reply:
It is related (because we need to display the pickup store name in the confirmation dialogue).
But still this is a good hint, on a second look I saw that this information can also be derived from the cart modification via entry.deliveryPointOfService.name, then the event does not need to be touched!